### DISCUSSION TITLE
What is the API behavior for Multi-SIM ?

### DISCUSSION THREAD

**Discussion by bigludo7** [camaraproject/DeviceSwap#10] [Q&A]
Discussion started in issue #6 
Proposal to continue the discussion here.

How multi-SIM affect the design?

@jgarciahospital 
> When a multi-SIM subscription is in place, more than one IMEI is associated to the same MSISDN. It is, depending on which device is active/connected, the record of IMEI associated tot he MSISDN will change time to time, sometimes more than one time per day/hour. Therefore, the device swap events will be triggered continuously and no valid information will be provided, since the event only represents that the user is activating one of his already existing devices.
Proposal is to only consider new devices as a valid device swap event, e.g. establishing a margin of 1 or several months in which a device can be used in the same MSISDN without considering it a device Swap event, as no valuable information for the API consumers.

@bigludo7 
> Probably we need to explore how 3GPP Monitoring event `CHANGE_OF_IMSI_IMEI_ASSOCIATION`  works for multi-SIM.

**Comment by jgarciahospital** [comment]
proposed to include this topic as general Commonalities issue in https://github.com/camaraproject/Commonalities/issues/302

In any case, agreement remains in treating this topic in the second version of this API.

**Comment by bigludo7** [comment]
Hi @jgarciahospital 
Say your proposal in Commonalities - make complete sense to have a first round of discussion at Commonalities level.
